.github/workflows/backend-test.yml

```diff
@@ -13,15 +13,45 @@ on:
       - 'hotfix/**'
       - 'release/**'
       - 'dependabot/**'
-    paths-ignore:
-      - 'frontend/**'
-      - '**.md'
+    paths:
+      - 'backend/**'
+      - '.github/workflows/backend*'
+      - 'docker-compose.yaml'
 
 jobs:
   backend-test:
     name: Test Backend
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
```

Comment:
I understand this is the user triggering the workflow. Does this need any setup for the developers?

Reply:
Nope, this is basically a user pre-configured by the action which has rights (for this repo) to write to this registry. We would need to setup username / passwords in GitHub secrets if we used a non-GitHub registry, I believe.